src/Utility/ExprFunctions.h

```diff
@@ -99,6 +99,8 @@ class EvalFuncSpatial : public RealFunc
     Scalar* z; //!< Z-coordinate
     Scalar* t; //!< Time
 
+    //! \brief Returns a const ref to a member.
+    //! !\param dir One-based index to member
     const Scalar& get(int dir) const
     {
```

Comment:
(This comment applies to Line 107 below): Isn't it proper to have the default: case as the last case and not the first? Guess it is because you want *x to be the default, but I also thought such fall-through case statements (without break or return before the next), generated annoying warnings in the later compilers. Or have they perhaps explicitly been deactivated in the build set-up.

Reply:
Empty fallthroughs do no warn. There is no proper or not about where you put the default label, but if you want I can of course put default at the end or duplicate the line.